components/inputs/input-date.js

```diff
@@ -169,14 +171,16 @@ class InputDate extends FormElementMixin(LocalizeCoreElement(LitElement)) {
 		const icon = (this.invalid || this._inputTextInvalid)
 			? html`<d2l-icon icon="tier1:alert" slot="left" style="${styleMap({ color: 'var(--d2l-color-cinnabar)' })}"></d2l-icon>`
 			: html`<d2l-icon icon="tier1:calendar" slot="left"></d2l-icon>`;
-		const tooltip = (this.validationError && !this._dropdownOpened) ? html`<d2l-tooltip align="start" announced for="${this._inputId}" state="error">${this.validationError}</d2l-tooltip>` : null;
+		const errrorTooltip = (this.validationError && !this._dropdownOpened) ? html`<d2l-tooltip align="start" announced for="${this._inputId}" state="error">${this.validationError}</d2l-tooltip>` : null;
+		const infoTooltip = (this._showInfoTooltip && !this.validationError && !this._inputTextInvalid && !this._dropdownOpened) ? html`<d2l-tooltip align="start" announced delay="1000" for="${this._inputId}">${this.localize(`${this._namespace}.openInstructions`, { format: shortDateFormat })}</d2l-tooltip>` : null;
```

Comment:
Could be slightly simplified: `(this._showInfoTooltip && !errorTooltip && !this._inputTextInvalid)`